src/prefect/server/models/task_runs.py

```diff
@@ -334,6 +335,54 @@ async def count_task_runs(
     return result.scalar()
 
 
+async def count_task_runs_by_state(
+    session: AsyncSession,
+    db: PrefectDBInterface,
+    flow_filter: Optional[schemas.filters.FlowFilter] = None,
+    flow_run_filter: Optional[schemas.filters.FlowRunFilter] = None,
+    task_run_filter: Optional[schemas.filters.TaskRunFilter] = None,
+    deployment_filter: Optional[schemas.filters.DeploymentFilter] = None,
+):
```

Comment:
Should we add a type hint about what gets returned here?

Reply:
def - done!